src/helpers/parseGSNError.ts

```diff
@@ -0,0 +1,15 @@
+function transformRelayErrorMessage(message: string) {
+  // Removes stack trace information
+  return message
+    .split('stack')
+    ?.filter((_, i) => i % 2 === 0)
+    ?.join('\n')
+}
+
+export default function (gSNError: string) {
+  const metaMaskMessage = 'MetaMask Message Signature:'
+  if (gSNError.includes(metaMaskMessage)) {
+    return gSNError.split(metaMaskMessage)[1]?.split(' stack:')[0]
```

Comment:
Can we extract this into function with proper name to give better understanding on what is going on here?

Reply:
sure